### PR TITLE
Issue 742/legend ng9

### PR DESCRIPTION
**Pull request by raitisbe**

### CODE REVIEW

File: components/print/partials/printdialog.html [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <div class="card panel-default hs-main-panel">
-    <hs-panel-header name="print" title="Print"></hs-panel-header>
+    <hs-panel-header name="print" i18n-title="Print"></hs-panel-header>
```
Comment (FilipLeitner) [inline components/print/partials/printdialog.html:2]: Might investigate a bit about the internationalization..later, seems like ngx-translate might be a way to go if we want to keep using .po files. https://phrase.com/blog/posts/best-libraries-for-angular-i18n/
Also not sure if its possible to support translations properly in hybrid app. Or does this built i18n way just works right away ?. Dont have an experience with these ng9 things ^^

**Comment by jmacura** [comment]
There are failing unit tests in the services which have been converted to classes.

**Comment by raitisbe** [comment]
> There are failing unit tests in the services which have been converted to classes.

Yeah, not sure if classes or because I changed the bootstraping process. I wIll investigate

File: components/print/partials/printdialog.html (hunk 1 above)
Comment (raitisbe) [reply components/print/partials/printdialog.html:2]: I was hoping i would work. I would expect that maybe just some module import would be necessary as is with Common module of @angular to tget ngIf and ngModel working. https://angular.io/guide/i18n

**Comment by raitisbe** [comment]
> There are failing unit tests in the services which have been converted to classes.

Tests are now working. Just took 2 full days ;)

**Pull request merged**